rbenv.d/exec/gem-rehash/rubygems_plugin.rb

```diff
@@ -10,27 +10,27 @@
   end
 end
 
-if defined?(Bundler::Installer) && Bundler::Installer.respond_to?(:install) && !Bundler::Installer.respond_to?(:install_without_rbenv_rehash)
+if defined?(Bundler::Installer) && Bundler::Installer.methods.include?(:install) && !Bundler::Installer.methods.include?(:install_without_rbenv_rehash)
```

Comment:
`Class.methods.include?` checks for the presence of a class method, but since we switched to monkeypatching `install` on the instance level, we should be checking method presence on the instance too

```suggestion
if defined?(Bundler::Installer) && Bundler::Installer.method_defined?(:install) && !Bundler::Installer.method_defined?(:install_without_rbenv_rehash)
```